rllib/agents/callbacks.py

```diff
@@ -196,7 +196,7 @@ def on_postprocess_trajectory(
         policy_id: PolicyID,
         policies: Dict[PolicyID, Policy],
         postprocessed_batch: SampleBatch,
-        original_batches: Dict[AgentID, SampleBatch],
+        original_batches: Dict[AgentID, Tuple[Policy, SampleBatch]],
```

Comment:
what, that's surprising ...
how can we mix these 2 things together?

Reply:
oh it's Tuple, not Union, I got scared.
Thanks!

Reply:
:) @gjoliver 

I think there is only very few places in RLlib where we mix different types e.g. in a return value (for example in the sampler code `_process_observations()`) and no, we probably shouldn't do this.